src/CMakeLists.txt

```diff
@@ -192,7 +202,13 @@ else(INTEL_SSE4_1)
 endif(INTEL_SSE4_1)
 
 
-set(EXTRALIBS uuid rt dl ${ATOMIC_OPS_LIBRARIES})
+set(EXTRALIBS rt ${ATOMIC_OPS_LIBRARIES})
+if(${LINUX})
+  set(LIB_DL dl)
+  set(LIB_RESOLV resolv)
+  set(LIB_UUID uuid)
+  set(EXTRALIBS "${EXTRALIBS} ${LIB_DL} LIB_UUID")
```

Comment:
@tchaikov 
Using APPEND adds `;` between the elements?
So that does not always work. 
It didn't work for the C_CFLAGS